airflow/contrib/hooks/spark_submit_hook.py

```diff
@@ -110,12 +111,19 @@ def __init__(self,
         self._num_executors = num_executors
         self._application_args = application_args
         self._verbose = verbose
-        self._sp = None
+        self._submit_sp = None
         self._yarn_application_id = None
 
         self._connection = self._resolve_connection()
         self._is_yarn = 'yarn' in self._connection['master']
 
+        self._should_track_driver_status = self._resolve_should_track_driver_status()
+        self._driver_id = None
+        self._driver_status = None
+
+    def _resolve_should_track_driver_status(self):
```

Comment:
how about adding some doc string here? something like 

"determines whether or not this hook should poll the spark driver status through subsequent spark-submit requests after the initial spark-submit request"